com.github.louis77.tuner.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "com.github.louis77.tuner",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.louis77.tuner",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
What does it need host access for?

Reply:
It didn't, thank you for the suggestion!